packages/react-hooks/src/useTreasury.ts

```diff
@@ -27,9 +27,13 @@ export function useTreasury (): Treasury {
   const [burn, setBurn] = useState<BN>();
 
   const treasuryBalance = useCall<DeriveBalancesAccount>(api.derive.balances.account, [TREASURY_ACCOUNT]);
-  const spendPeriod = api.consts.treasury.spendPeriod;
+  const spendPeriod = api.consts.treasury ? api.consts.treasury.spendPeriod : api.createType('BlockNumber', 0);
```

Comment:
```suggestion
  const spendPeriod = api.consts.treasury ? api.consts.treasury.spendPeriod : BN_ZERO;
```

Just the BN_ZERO import from `@polkadot/util` (since we don't rely on it being an actual BlockNumber, just a BN, all fine). Generally try to steer clear of `createType` unless _really_ needed.